gnmpagerduty/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models import Model,BigIntegerField,IntegerField,CharField,DateTimeField
+
+
+class StorageData(Model):
+    storage_id = CharField(max_length=64,unique=True)
+    maximum_size = BigIntegerField(blank=True,null=True)
+    current_size = BigIntegerField(blank=True,null=True)
+    trigger_size = BigIntegerField(blank=True,null=True)
+    incident_key = CharField(max_length=64,blank=True, default='')
```

Comment:
a foreign key is a reference to a specific record in another data model, i.e. a specific row in another table.

please remove them from the first model and make a migration to that effect if they are not used.